osu.Game/Screens/Multi/Match/Components/MatchLeaderboard.cs

```diff
@@ -50,7 +50,7 @@ protected override APIRequest FetchScores(Action<IEnumerable<APIUserScoreAggrega
 
         protected override LeaderboardScore CreateDrawableScore(APIUserScoreAggregate model, int index) => new MatchLeaderboardScore(model, index);
 
-        protected override LeaderboardScore CreateDrawableTopScore(APIUserScoreAggregate model) => new MatchLeaderboardScore(model, 0);
+        protected override LeaderboardScore CreateDrawableTopScore(APIUserScoreAggregate model) => new MatchLeaderboardScore(model, model.Position ?? 0);
```

Comment:
Not sure about this coercion to 0 - it doesn't really ever make sense to have a 0 position. Would probably rather prefer just outright throwing, or being able to handle a `null` position and hiding the position text instead.

Reply:
Sure, the base class already handled nulls.